eras/conway/impl/src/Cardano/Ledger/Conway/PParams.hs

```diff
@@ -65,21 +232,394 @@ instance Crypto c => EraPParams (ConwayEra c) where
 
 instance Crypto c => AlonzoEraPParams (ConwayEra c) where
   hkdCoinsPerUTxOWordL = notSupportedInThisEraL
-  hkdCostModelsL = lens bppCostModels $ \pp x -> pp {bppCostModels = x}
-  hkdPricesL = lens bppPrices $ \pp x -> pp {bppPrices = x}
+  hkdCostModelsL = lens cppCostModels $ \pp x -> pp {cppCostModels = x}
+  hkdPricesL = lens cppPrices $ \pp x -> pp {cppPrices = x}
   hkdMaxTxExUnitsL :: forall f. HKDFunctor f => Lens' (PParamsHKD f (ConwayEra c)) (HKD f ExUnits)
   hkdMaxTxExUnitsL =
-    lens (hkdMap (Proxy @f) unOrdExUnits . bppMaxTxExUnits) $ \pp x ->
-      pp {bppMaxTxExUnits = hkdMap (Proxy @f) OrdExUnits x}
+    lens (hkdMap (Proxy @f) unOrdExUnits . cppMaxTxExUnits) $ \pp x ->
+      pp {cppMaxTxExUnits = hkdMap (Proxy @f) OrdExUnits x}
   hkdMaxBlockExUnitsL :: forall f. HKDFunctor f => Lens' (PParamsHKD f (ConwayEra c)) (HKD f ExUnits)
   hkdMaxBlockExUnitsL =
-    lens (hkdMap (Proxy @f) unOrdExUnits . bppMaxBlockExUnits) $ \pp x ->
-      pp {bppMaxBlockExUnits = hkdMap (Proxy @f) OrdExUnits x}
-  hkdMaxValSizeL = lens bppMaxValSize $ \pp x -> pp {bppMaxValSize = x}
+    lens (hkdMap (Proxy @f) unOrdExUnits . cppMaxBlockExUnits) $ \pp x ->
+      pp {cppMaxBlockExUnits = hkdMap (Proxy @f) OrdExUnits x}
+  hkdMaxValSizeL = lens cppMaxValSize $ \pp x -> pp {cppMaxValSize = x}
   hkdCollateralPercentageL =
-    lens bppCollateralPercentage $ \pp x -> pp {bppCollateralPercentage = x}
+    lens cppCollateralPercentage $ \pp x -> pp {cppCollateralPercentage = x}
   hkdMaxCollateralInputsL =
-    lens bppMaxCollateralInputs $ \pp x -> pp {bppMaxCollateralInputs = x}
+    lens cppMaxCollateralInputs $ \pp x -> pp {cppMaxCollateralInputs = x}
 
 instance Crypto c => BabbageEraPParams (ConwayEra c) where
-  hkdCoinsPerUTxOByteL = lens bppCoinsPerUTxOByte (\pp x -> pp {bppCoinsPerUTxOByte = x})
+  hkdCoinsPerUTxOByteL = lens cppCoinsPerUTxOByte (\pp x -> pp {cppCoinsPerUTxOByte = x})
+
+instance Crypto c => ConwayEraPParams (ConwayEra c) where
+  hkdVotingThresholdsL = lens cppVotingThresholds (\pp x -> pp {cppVotingThresholds = x})
+  hkdMinCCSizeL = lens cppMinCCSize (\pp x -> pp {cppMinCCSize = x})
+  hkdCCTermLimitL = lens cppCCTermLimit (\pp x -> pp {cppCCTermLimit = x})
+  hkdGovExpirationL = lens cppGovExpiration (\pp x -> pp {cppGovExpiration = x})
+  hkdGovDepositL = lens cppGovDeposit (\pp x -> pp {cppGovDeposit = x})
+  hkdDRepDepositL = lens cppDRepDeposit (\pp x -> pp {cppDRepDeposit = x})
+  hkdDRepActivityL = lens cppDRepActivity (\pp x -> pp {cppDRepActivity = x})
+
+instance Crypto c => EraGovernance (ConwayEra c) where
```

Comment:
Is it better for this to reside in `PParams` instead of `Governance` module?

Reply:
This was moved due to a cyclical import problem that I started discussing on Slack. This PR is on hold now, until the module for `Conway.Governance` is split properly, in a separate PR.